BUILD.bazel

```diff
@@ -139,13 +135,28 @@ py_library(
 
 py_wheel(
     name = "opensearch_protos_wheel",
-    distribution = "opensearch-protos",
+    distribution = "opensearch-protobufs",
     python_tag = "py3",
     version = "0.10.0",
     requires = [
         "protobuf>=3.20.3",
         "grpcio>=1.68.1",
     ],
+    author = "OpenSearch",
+    license = "Apache-2.0",
+    classifiers = [
+        "Intended Audience :: Developers",
+        "License :: Apache-2.0 Software License",
+        "Programming Language :: Python :: 3",
+    ],
+    homepage = "https://opensearch.org/",
+    project_urls = {
+        "Bug Tracker": "https://github.com/opensearch-project/opensearch-protobufs/issues",
+        "Documentation": "https://github.com/opensearch-project/opensearch-protobufs/blob/main/README.md",
+        "Source Code": "https://github.com/opensearch-project/opensearch-protobufs",
+    },
+    platform = "any",
+    python_requires = ">=3.8",
```

Comment:
Python 3.8 is already deprecated and 3.9 will be out of support in 2 months.
Considering moving to newer version such as 3.10?
https://devguide.python.org/versions/